### DISCUSSION TITLE
SMTP configuraiton doesnt work

### DISCUSSION THREAD

**Discussion by ankwad** [chatwoot/chatwoot#5391] [Self hosted]
Hi 
I have tried multiple settings.. 

Is there a way to chek whats wrong.. where can check the logs

**Comment by vishnu-narayanan** [comment]
Check the Chatwoot worker(sidekiq) logs.

**Reply by ankwad** [reply]
Well the SMTP configuration just doesnt work ..  
Keep getting server 500 error
Can someone guide as to how to go about

**Reply by ankwad** [reply]
A way to figure out is pinging via telnet your SMTP mail server along with the port.

port 465 seems not to be working.. atleast not with the custom installation, something more needs to be done in Ubuntu.

I got it to work with port 587 after doing telnet check 

hope it helps someone ...
